CrewChiefV4/NoisyCartesianCoordinateSpotter.cs

```diff
@@ -414,6 +414,80 @@ public void clearState()
             }
         }
 
+        /*
+            public enum CarLeftRight : uint
+    {
+        irsdk_LROff,
+        irsdk_LRClear, // no cars around us.
```

Comment:
Why is this not in iRacing spotter, but in NoisyCartesianCoordinateSpotter?  I believe this class should be game agnostic.

Reply:
I think I understand it mostly though.  It is because we don't use game positions but use data about cars around us directly from the game.  In this case it is probably ok :)  just wanted to keep class cleaner.

Reply:
Its part of the c++ official iRacing SDK but its not included in the c# version as its a bit outdated and missing some varibles.

Reply:
what I was trying to suggest, that maybe it is better to keep function:
   public void triggerInternal(int carLeftRight)

In iRacing spotter class, because this function is not related to calculating spotter messages based on 2D positions.  Just a suggestion.